src/components/ha-cover-controls.js

```diff
@@ -12,7 +12,7 @@ class HaCoverControls extends PolymerElement {
           white-space: nowrap;
         }
         [invisible] {
-          visibility: hidden !important;
```

Comment:
So the reason we used invisible is so that controls don't jump around and are aligned if you have multiple covers in multiple row.s

Reply:
I thought that as well but assumed that you agreed with the change given that you added it to the frontend Todo project

Reply:
@balloob so should we close this and the issue then?

Reply:
I think that it's fine to hide them if the feature is not supported, as it will never show up. 

I think that if it's not available, we should mark it as disabled.

Reply:
Not currently available?

Reply:
If the cover is open, the "open/up" button should be disabled as it's not currently available.

Reply:
Ah, yes. Got it.

Reply:
Controls are already disabled based on state, so I believe there is nothing more to do here. Closing PR and issue